modelmapper/base.py

```diff
@@ -159,10 +157,8 @@ def _get_all_values_per_clean_name(self, path):
                         raise ValueError("Your data might have new lines in the field names. "
                                          "Please fix that and try again.")
                     else:
-                        result[field_name].append(v)
-
-        for field_name in self.settings.fields_to_be_scrubbed:
-            result[field_name] = [''] * len(result[field_name])
+                        if field_name not in self.settings.fields_to_be_scrubbed:
```

Comment:
In this new code, scrubbing actually removes the field instead of just setting its value to empty string.